crawl-ref/source/godabil.cc

```diff
@@ -4242,48 +4244,85 @@ bool gozag_setup_potion_petition(bool quiet)
     return true;
 }
 
-bool gozag_potion_petition()
+// What's the price of a particular potion effect?
+int _gozag_price_effect(item_def potion)
```

Comment:
@alexjurkiewicz I think you just need to add a `static` to the front: `static int ...`